src/network/gun-servers.ts

```diff
@@ -1,2 +1,2 @@
-export const gunServers = ['https://gun.maskbook.com/gun'] as const
+export const gunServers = ['https://hk.gun.dimension.chat/gun', 'https://gun.maskbook.com/gun'] as const
```

Comment:
This will slow down the gun. Gun will ensure the data do sync to both two server instead of randomly choose one and let the server sync by themself

Reply:
by design?
https://github.com/amark/gun/issues/309

Reply:
![image](https://user-images.githubusercontent.com/3842474/82173276-37e41b80-98ff-11ea-9784-c6ace3c4de75.png)

Reply:
random selected a server?

Reply:
have no idea